transport-msg.tex

```diff
@@ -982,21 +1036,31 @@ \subsubsection{Overview}
 
```

Comment:
Maybe we can keep the order of the fields same as GET_CONFIG ? i.e. offset, size, generation, data.

Reply:
OK but I would vote we change GET_CONFIG to match this (gen, offset, size, data)
gen is always good info but often offset & size are 0 and data is empty